src/tool/cpp/cppwinrt/main.cpp

```diff
@@ -32,9 +32,7 @@ namespace xlang
         { "base", 0, 0, {}, "Generate base.h unconditionally" },
         { "opt", 0, 0, {}, "Generate component projection with unified construction support" },
         { "help", 0, cmd::option::no_max, {}, "Show detailed help with examples" },
-        { "root", 0, 1 }, // "Root folder name for projection headers (defaults to winrt)
         { "filter" }, // One or more prefixes to include in input (same as -include)
-        { "lib", 0, 1 }, // Specify library prefix (defaults to winrt)
```

Comment:
I'll fix this up. I forgot the lib prefix applies to those module exports. It's critical that they can be changed to allow multiple libs to be combined into a single DLL.